kbase/narrative
# DATAUP-468 verify files on load

# Description of PR purpose/changes

The goal is to verify that staging area files still exist when loading a bulk import cell. It should show errors and prevent app starting if they're gone.

This is part 1 of a multi-part PR. There's some progress here, but some cases won't be handled yet until the other PRs are done. I just wanted to keep them smallish.

This PR does the following:
1. Fixes select2 styling to show an error when something bad is selected.
2. Adds an option to select boxes of invalid selections on start. Selecting these will force the error to happen.
3. Disables a (I think?) redundant parameter check on start. The BI cell was doing a global parameter check whenever it was started and used that to figure out what state it should be in. However, that state should be saved already, so this might be redundant.

Next PR will (hopefully) speed things up by making a front end cache for staging file information. It can get sluggish with multiple bulk import cells with lots of file data to fetch. Maybe it's worth holding off, though, and optimizing later.


# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-468

# Testing Instructions
* Details for how to test the PR:
1. Make a Bulk Import cell with some files
2. Get those files selected
3. Save the Narrative (probably easiest to do that at this step)
4. Delete one of the files.
5a. Reset the configure tab (i.e. disable and re-enable the tab, or hit the info tab and go back, or toggle to a different data type and go back) and see the error.
5b. Reload the narrative to see the error.

# Dev Checklist:
